src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -83,11 +95,25 @@ public CommandResult.Error getCommandResultError(String message, Response.Status
     }
     DebugModeConfig debugModeConfig = config.getConfigMapping(DebugModeConfig.class);
     final boolean debugEnabled = debugModeConfig.enabled();
-    final Map<String, Object> fields =
-        debugEnabled
-            ? Map.of(
-                "errorCode", errorCode.name(), "exceptionClass", this.getClass().getSimpleName())
-            : Map.of("errorCode", errorCode.name());
+    final boolean extendError = config.getConfigMapping(OperationsConfig.class).extendError();
+    Map<String, Object> fields = null;
+    if (debugEnabled) {
```

Comment:
wondering about the logic here. Where is "debugEnabled" set?

Reply:
Something seems strange to me about this. We're using two different booleans to pick from one of three different cases. Is it possible that debugEnabled and extendError could both be set? Which one should take precedence (right now it is debugEnabled).

Reply:
rethought the logic here and changed my code to 
```
  if (extendError) {
    fields =
        Map.of(
            "errorCode",
            errorCode.name(),
            "family",
            errorFamily,
            "scope",
            errorScope,
            "title",
            title);
  } else {
    fields = Map.of("errorCode", errorCode.name());
  }

  if (debugEnabled) {
    fields.put("exceptionClass", this.getClass().getSimpleName());
  }
```
It will first decide the basic error structure based on `extendError` boolean and add `exceptionClass` if debug mode is enabled. WDYT?

Reply:
This looks cleaner / less confusing to me, thanks @Hazel-Datastax